studio/components/layouts/ProjectLayout/LayoutHeader/HelpPopover.tsx

```diff
@@ -39,11 +40,18 @@ const HelpPopover: FC<Props> = () => {
                 hosted services.
               </p>
             </Typography.Text>
-            <Link href={supportUrl}>
-              <Button className="sbui-default-button--dark-white" icon={<IconMail />}>
-                Contact support team
-              </Button>
-            </Link>
+            <div className='flex space-x-1'>
+              <Link href={supportUrl}>
+                <Button className="sbui-default-button--dark-white" size="tiny" icon={<IconMail />}>
+                  Contact support
+                </Button>
+              </Link>
+              <Link href="supabase.com/docs/">
+                <Button type="secondary" size="tiny" icon={<IconBookOpen />}>
+                  Documentation
```

Comment:
```suggestion
                  Docs
```

I think just Docs could be better here to give the buttons more room